monkestation/code/modules/cybernetics/augments/arm_augments/unsorted.dm

```diff
@@ -143,17 +143,15 @@
 			return
 		H.cybernetics_ammo[zone] = null
 
-		counter_ref.hud = null
```

Comment:
In case you're thinking "This looks odd!" - this doesn't actually change anything. Right below this (now-removed) line is `QDEL_NULL(counter_ref)` - and `counter_ref`'s type will automatically null out `.hud` upon deletion.